packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+## 2.7.0
+* Revert ProgramExplorer changes [#3417](https://github.com/flutter/devtools/pull/3417)
```

Comment:
nit: maybe clear ProgramExplorer from the changelog for this release as it is in and then out.

Reply:
done